analyzer/src/main/kotlin/Analyzer.kt

```diff
@@ -143,35 +141,14 @@ class Analyzer(private val config: AnalyzerConfiguration) {
                             val results = manager.resolveDependencies(files)
 
                             // By convention, project ids must be of the type of the respective package manager.
-                            results.forEach { (_, result) ->
+                            results.onEach { (_, result) ->
```

Comment:
Is it intentionally part of this change?

Reply:
Yes, it's needed to return `results` to the enclosing `withContext` (`forEach` just returns `Unit`).